src/random/isaac.cr

```diff
@@ -12,21 +12,23 @@ class Random::ISAAC
   private getter bb
   private getter cc
 
-  private def self.random_seeds
-    (uninitialized StaticArray(UInt32, 8)).tap do |seeds|
-      System::Random.random_bytes(seeds.to_slice)
-    end
+  private alias Seeds = StaticArray(UInt32, 8)
+
+  private def random_seeds
+    result = uninitialized Seeds
+    SecureRandom.random_bytes(result.unsafe_as(StaticArray(UInt8, sizeof(Seeds))).to_slice)
```

Comment:
Please extract `result.unsafe_as(StaticArray(UInt8, sizeof(Seeds))).to_slice` to a `result_slice` variable. The line's getting quite messy.